# pyre-ignore-all-errors
def f():
    try:
        a
    finally:
        return 42
